tests/test_image_to_entries.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from khoj.processor.content.images.image_to_entries import ImageToEntries
+
+
+def test_png_to_jsonl():
+    with open("tests/data/images/testocr.png", "rb") as f:
+        image_bytes = f.read()
+    data = {"tests/data/images/testocr.png": image_bytes}
+    entries = ImageToEntries.extract_image_entries(image_files=data)
+    assert len(entries) == 2
+    assert len(entries[1]) == 31
+
+
+def test_jpg_to_jsonl():
+    with open("tests/data/images/nasdaq.jpg", "rb") as f:
+        image_bytes = f.read()
+    data = {"tests/data/images/nasdaq.jpg": image_bytes}
+    entries = ImageToEntries.extract_image_entries(image_files=data)
+    assert len(entries) == 2
+    assert len(entries[1]) == 143
```

Comment:
Why are there so many `entries[1]`? Is this counting the length of the OCR-ed string?

Reply:
Those are the number of entries generated from the images. Perhaps we can condense it down a little bit

Reply:
Oh why so many entries? Does the test image have a lot of text?

Reply:
not quite. It could just be the way rapidocr splits the text

Reply:
I can probably chunk it together or something.

Reply:
Yeah joining the string may make sense if it seems like RapidOCR is splitting the text in a non-sensical way

Reply:
Addressed this by taking it all as one blurb and joining the individual chunks with spaces. Will update corresponding unit tests.